trunk/Admin/js/admin-quickbar-admin.js

```diff
@@ -125,12 +125,12 @@
  * Set localStorage
  */
 function setPostListStorage() {
```

Comment:
Rethink the naming. A `set`-method is supposed to accept a value and set a private member. That's not the case here.

Reply:
Renamed to refreshPostListStorage, but i´m currently not sure about this. I´m thinking about reinit or init, but don´t like it.